### PR TITLE
fix: Continue remove account_id in config

### PR DESCRIPTION
**Pull request by bbernays**
- [ ] Relates OR Closes #0000
- [ ] Added integration tests as described [here](https://docs.cloudquery.io/docs/developers/sdk/testing)

Output from integration tests:
More information about running the tests [here](../docs/contributing/e2e_tests.md)

```
$ go test -tags=integration ./...

```

### CODE REVIEW

File: client/config.go [hunk 1]
```diff
@@ -21,11 +21,11 @@ type Config struct {
 func (c Config) Example() string {
 	return ` configuration {
   // Optional. if you want to assume role to multiple account and fetch data from them
-  // accounts "<YOUR ACCOUNT ID>" {
+  // accounts "<UNIQUE ACCOUNT IDENTIFIER>" {
     // Optional. Role ARN we want to assume when accessing this account
     // role_arn = < YOUR_ROLE_ARN >
-    // Optional. Account ID we want to assume when accessing this account - override the block label
-    // account_id = < YOUR ACCOUNT ID >
```
Comment (roneli) [inline client/config.go:28]: why? its still a valid attribute in the hcl, I am confused from the previous PR #396

File: client/testdata/assume_role.hcl [hunk 2]
```diff
@@ -17,7 +17,6 @@ provider "aws" {
 
     accounts "default" {
       role_arn = "${ASSUME_ROLE_ARN}"
-      account_id = "${ASSUME_ROLE_ACCOUNT_ID}"
```
Comment (roneli) [inline client/testdata/assume_role.hcl:20]: mistake?

**Review by roneli — CHANGES_REQUESTED** [review]

File: client/config.go (hunk 1 above)
Comment (bbernays) [reply client/config.go:28]: The mistake is that it is still a valid attribute in hcl. fixed now

Comment (bbernays) [reply client/config.go:28]: We had already made it an error: https://github.com/cloudquery/cq-provider-aws/blob/2fc0a4d434fb274352cb1c74562b22fc8adb8713/client/client.go#L361-L363

File: client/testdata/assume_role.hcl (hunk 2 above)
Comment (bbernays) [reply client/testdata/assume_role.hcl:20]: No

**Review by roneli — APPROVED** [review]

**Pull request merged**